template/.envrc

```diff
@@ -1,7 +1,12 @@
+#!/usr/bin/env bash
+# ^ added for shellcheck and file-type detection
+
 if [[ $(type -t use_flake) != function ]]; then
   echo "ERROR: use_flake function missing."
   echo "Please update direnv to v2.30.0 or later."
   exit 1
 fi
-
 use flake
+
+# Watch & reload direnv on change
+watch_file devshell.toml
```

Comment:
Can you move the watch_file before the use_flake? That way, reloads will happen if there is something broken in the devshell.toml.

Reply:
Good call :thumbs_up: